src/models/VecchiaModel.jl

```diff
@@ -7,17 +7,18 @@ function VecchiaModel(::Type{S}, iVecchiaMLE::VecchiaMLEInput) where {S<:Abstrac
     ncon::Int = length(cache.colptrL) - 1
 
     # Allocating data    
-    x0_::S = fill!(S(undef, nvar), zero(T))
-    y0::S = fill!(S(undef, ncon), zero(T))
+    x0_::S  = fill!(S(undef, nvar), zero(T))
+    y0::S   = fill!(S(undef, ncon), zero(T))
     lcon::S = fill!(S(undef, ncon), zero(T))
     ucon::S = fill!(S(undef, ncon), zero(T))    
-    lvar::S = fill!(S(undef, nvar), -Inf)
-    uvar::S = fill!(S(undef, nvar), Inf)
+    lvar::S = fill!(S(undef, nvar), -T(1e10))
+    uvar::S = fill!(S(undef, nvar),  T(1e10))
```

Comment:
We can use that as default but sure to modify the `z` part of the vector of variables `w`.